### PR TITLE
fix convert

### PR DESCRIPTION
**Pull request by wonderflow**
## Changes
   
- [ ] feature1
- [ ] feature2
- [ ] fixbug1
- [ ] fixbug2
 
## Reviewers

- [ ] @[someone] please review
- [ ] @[someotherone] please review

## Wiki Changes
 
- options1...
- options2...
    
## Checklist
   
- [ ] Rebased/mergeable
- [ ] Tests pass
- [ ] Wiki updated

### CODE REVIEW

File: transforms/mutate/convert.go [hunk 1]
```diff
@@ -341,13 +346,17 @@ func dataConvert(data interface{}, schema DslSchemaEntry) (converted interface{}
 		case reflect.Float32, reflect.Float64:
 			return int64(value.Float()), nil
 		case reflect.String:
-			if converted, err = strconv.ParseInt(value.String(), 10, 64); err == nil {
-				return
+			val := value.String()
+			val = strings.TrimSpace(val)
```
Comment (Unknwon) [inline transforms/mutate/convert.go]: 为什么不直接 `strings.TrimSpace(value.String())` 呢

File: transforms/mutate/convert.go [hunk 2]
```diff
@@ -359,7 +368,12 @@ func dataConvert(data interface{}, schema DslSchemaEntry) (converted interface{}
 		case reflect.Float32, reflect.Float64:
 			return data, nil
 		case reflect.String:
-			if converted, err = strconv.ParseFloat(value.String(), 10); err == nil {
+			val := value.String()
+			val = strings.TrimSpace(val)
```
Comment (Unknwon) [inline transforms/mutate/convert.go]: 同上

File: transforms/mutate/convert.go [hunk 3]
```diff
@@ -359,7 +368,12 @@ func dataConvert(data interface{}, schema DslSchemaEntry) (converted interface{}
...
+			if val == "" {
+				return nil, nil
+			}
+			if converted, err = strconv.ParseFloat(val, 10); err == nil {
 				return
```
Comment (Unknwon) [inline transforms/mutate/convert.go]: 这里 `return nil, err` 吧

File: transforms/mutate/convert.go (hunk 1 above)
Comment (wonderflow) [reply transforms/mutate/convert.go]: val下面还要用

File: transforms/mutate/convert.go (hunk 2 above)
Comment (wonderflow) [reply transforms/mutate/convert.go]: 同上

File: transforms/mutate/convert.go (hunk 3 above)
Comment (wonderflow) [reply transforms/mutate/convert.go]: 这个是 err==nil的时候返回

File: transforms/mutate/convert.go (hunk 1 above)
Comment (Unknwon) [reply transforms/mutate/convert.go]: 我是指 `val := strings.TrimSpace(value.String())`

**Pull request merged**